### DISCUSSION TITLE
🌟 Announcing the revitalization of scikit-bio! 🌟

### DISCUSSION THREAD

**Discussion by qiyunzhu** [scikit-bio/scikit-bio#1935] [Announcements]
We are excited to announce the resurgence of **scikit-bio** (https://scikit.bio)! With a re-assembled developer team and funding from the DOE, we're back with renewed vigor in 2024!

Our vision is to expand scikit-bio into a more robust and versatile library, catering to the ever-growing demands of multi-omic data analysis. This resurgence marks a new chapter in which we will focus on:

- Streamlining the analysis of diverse, massive omic data, emphasizing efficiency and versatility.
- Integrating advanced techniques for multi-omic integration to unravel the complex interplay between biological systems and environments.
- Implementing methods for modeling and annotating biological features utilizing community ecology and phylogenetics.

We invite the scientific community to join us in shaping the future of scikit-bio. Your expertise, feedback, and contributions will be the driving force behind this exciting phase.

Stay [tuned for updates](https://github.com/scikit-bio/scikit-bio/discussions/categories/announcements), and let's innovate together for a deeper understanding of bio-complexities!

## Get involved with scikit-bio

Your questions, ideas, and contributions matter!

Join our community on [GitHub Discussions](https://github.com/scikit-bio/scikit-bio/discussions): This is your go-to place for asking questions, sharing insights, and participating in discussions about scikit-bio. Engage with both the developers and fellow users here.

Report issues and bugs: If you encounter specific problems when using scikit-bio, let us know directly through the [GitHub Issues](https://github.com/scikit-bio/scikit-bio/issues) page. Your reports are vital for the continuous improvement of scikit-bio.

Wanna contribute? We enthusiastically welcome community contributors! Whether it's adding new features, improving code, or enhancing documentation, your contributions drive scikit-bio and open-source bioinformatics forward. Start your journey by reading the [Contributor's guidelines](https://scikit.bio/contribute.html).

**Comment by auvipy** [comment]
that's great! how to join the team?

**Reply by qiyunzhu** [reply]
Our team welcome you 😊 Check out the [quick start guide](https://scikit.bio/contribute.html#getting-started) for contributors. We are in the process of restructuring the codebase and documentation, so be prepared for some exciting changes ahead. If you are interested in contributing a specific feature set, or would like to discuss our "features wanted" list and brainstorm together, email me at qiyunzhu@gmail.com and let's chat!
